src/CreditCard.vue

```diff
@@ -31,8 +32,9 @@
                     <label for="card-number">{{ trans.card.label}}</label>
                     <input type="text"
                         name="card_number"
+                           id="card-number"
                         ref="cardNumber"
-                        pattern="[0-9]*"
+                        pattern="[0-9 ]*"
```

Comment:
why the extra space after the 9?

Reply:
Mask adds a space to card number, pattern doesn't allow it.